[2024-06-26T14:25:29.821+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-26T14:25:29.834+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: windturbine.group_database.insert_data scheduled__2024-06-26T14:21:00+00:00 [queued]>
[2024-06-26T14:25:29.839+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: windturbine.group_database.insert_data scheduled__2024-06-26T14:21:00+00:00 [queued]>
[2024-06-26T14:25:29.839+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-26T14:25:29.846+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): group_database.insert_data> on 2024-06-26 14:21:00+00:00
[2024-06-26T14:25:29.851+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=128) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-26T14:25:29.852+0000] {standard_task_runner.py:63} INFO - Started process 132 to run task
[2024-06-26T14:25:29.852+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'windturbine', 'group_database.insert_data', 'scheduled__2024-06-26T14:21:00+00:00', '--job-id', '533', '--raw', '--subdir', 'DAGS_FOLDER/windturbine.py', '--cfg-path', '/tmp/tmpp7fsvuwp']
[2024-06-26T14:25:29.853+0000] {standard_task_runner.py:91} INFO - Job 533: Subtask group_database.insert_data
[2024-06-26T14:25:29.878+0000] {task_command.py:426} INFO - Running <TaskInstance: windturbine.group_database.insert_data scheduled__2024-06-26T14:21:00+00:00 [running]> on host 98350b7a6ca8
[2024-06-26T14:25:29.952+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='liviataina.ltab@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='windturbine' AIRFLOW_CTX_TASK_ID='group_database.insert_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-26T14:21:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-26T14:21:00+00:00'
[2024-06-26T14:25:29.952+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-26T14:25:29.959+0000] {sql.py:276} INFO - Executing: 
        INSERT INTO sensors (idtemp, powerfactor, hydraulicpressure, temperature, timestamp)
        VALUES (%(idtemp)s, %(powerfactor)s, %(hydraulicpressure)s, %(temperature)s, %(timestamp)s)
        ON CONFLICT (idtemp)
        DO UPDATE SET
            powerfactor = EXCLUDED.powerfactor,
            hydraulicpressure = EXCLUDED.hydraulicpressure,
            temperature = EXCLUDED.temperature,
            timestamp = EXCLUDED.timestamp;
    
[2024-06-26T14:25:29.963+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-06-26T14:25:29.967+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-06-26T14:25:29.969+0000] {sql.py:487} INFO - Running statement: 
        INSERT INTO sensors (idtemp, powerfactor, hydraulicpressure, temperature, timestamp)
        VALUES (%(idtemp)s, %(powerfactor)s, %(hydraulicpressure)s, %(temperature)s, %(timestamp)s)
        ON CONFLICT (idtemp)
        DO UPDATE SET
            powerfactor = EXCLUDED.powerfactor,
            hydraulicpressure = EXCLUDED.hydraulicpressure,
            temperature = EXCLUDED.temperature,
            timestamp = EXCLUDED.timestamp;
    , parameters: {'idtemp': '1', 'powerfactor': '0.8837929080361997', 'hydraulicpressure': '78.86011124702158', 'temperature': '20.279809506572597', 'timestamp': '2023-03-19 17:26:55.230351'}
[2024-06-26T14:25:29.970+0000] {sql.py:496} INFO - Rows affected: 1
[2024-06-26T14:25:29.972+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-26T14:25:29.977+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=windturbine, task_id=group_database.insert_data, run_id=scheduled__2024-06-26T14:21:00+00:00, execution_date=20240626T142100, start_date=20240626T142529, end_date=20240626T142529
[2024-06-26T14:25:30.024+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-26T14:25:30.052+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-26T14:25:30.054+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
